src/OpenProject.Shared/BcfApi/Extensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using iabi.BCF.APIObjects.V21;
+using OpenProject.Shared.Math3D;
+
+namespace OpenProject.Shared.BcfApi
+{
+  public static class Extensions
```

Comment:
That's a very generic name, maybe call it `Vector3Extensions`?

Reply:
Good point, will try to figure a better summary for the extensions. Focus in this class should be extensions using the bcf api, so it is not only about Vector3

Reply:
using directly `BcfApiExtensions`? first I thought, the namespace should be enough to set the context, but it is a small step for me, if it improves readability in your eyes.

Reply:
`BcfApiExtensions` sounds good to me😀